cirq-core/cirq/protocols/json_test_data/SymmetricalQidPair.json

```diff
@@ -1,13 +0,0 @@
-{
```

Comment:
I don't think we're ever supposed to break loading old json data.

Consider loading as a tuple of two qubits and raising a warning

Reply:
Changed to be consistent with other "non breaking old loading behavior".